src/entities/metadata/metadata.mock.ts

```diff
@@ -63,6 +64,7 @@ export const mockMetadataData = (): TMetadata[] => [
 		id: '2',
 		title: 'Test metadata',
 		description: 'this is a very long description for test metadata',
+		source: 'http://testurl.com',
```

Comment:
source bestaat al in de test data
![image](https://github.com/user-attachments/assets/6eec6280-2ad9-4c5b-9c43-fc3dcc2f1e09)